keyboards/telophase/readme.md

```diff
@@ -0,0 +1,24 @@
+Mitosis
```

Comment:
Pretty sure that this isn't the Mitosis. ;)

If you're using the same code for the receiver, that's fine. But please update the information to be accurate here.